src/controller/InvokeInteraction.h

```diff
@@ -98,7 +98,7 @@ InvokeCommandRequest(Messaging::ExchangeManager * exchangeMgr, SessionHandle ses
                      uint16_t timedInvokeTimeoutMs)
 {
     return InvokeCommandRequest(exchangeMgr, sessionHandle, endpointId, requestCommandData, onSuccessCb, onErrorCb,
-                                timedInvokeTimeoutMs);
+                                MakeOptional(timedInvokeTimeoutMs));
```

Comment:
This seems like a footgun ... maybe consider different names for functions insted of relying on overloading.